kratos/utilities/exact_mortar_segmentation_utility.h

```diff
@@ -76,19 +79,26 @@ namespace Kratos {
  * @brief This utility calculates the exact integration necessary for the Mortar Conditions
  * @details The utility performs a mortar segmentation in order to obtain the exact integration of the geometry intersected
  * @author Vicente Mataix Ferrandiz
+ * @tparam TDim The dimension of work
+ * @tparam TNumNodes The number of nodes of the slave
+ * @tparam TBelong If we consider belonging of nodes or not
```

Comment:
I know I asked this before, but I forgot what it means
maybe you can enhance the description?

Reply:
When you do the intersections in order to get the directional derivatives you need to know where the intersections belongs to calculate the derivatives. This says between whcih nodes the intersection belongs